src/clarity-angular/datagrid/datagrid-action-overflow.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component, EventEmitter, HostListener, Input, Output, ElementRef} from "@angular/core";
+import "../../clarity-icons/shapes/essential-shapes";
```

Comment:
Just discussed with @Shijir. We're actually supposed to package all icons clarity components depend on into core (as opposed to essential). He's going to move ellipses into core in #455. Will remove this and revert the change in the bundle task as well.